Source: http://lxde.svn.sourceforge.net/viewvc/lxde/trunk/lxappearance/configure.in?r1=1817&r2=1823
Upstream: ^^^
Reason: solved the problem on runs configure in infinite loop
--- lxappearance/configure.in	2009/07/14 11:49:21	1817
+++ lxappearance/configure.in	2009/07/15 12:40:26	1823
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC
